basics/installation/install-from-cli.md

```diff
@@ -0,0 +1,111 @@
+| `engine`        | Engine for MySQL                           | InnoDB                       | InnoDB, MyISAM                                                                                        |
+| `name`          | Name of the shop                           | PrestaShop                   | _string_                                                                                              |
+| `activity`      | Default activity of the shop               | 0                            | Id of an activity ([Complete list of activities][activities])                                         |
+| `country`       | Country of the shop                        | fr                           | 2 letters Alpha-2 code of ISO-3166 list([ISO-3166][iso-3166])                                         |
+| `firstname`     | Admin user firstname                       | John                         | _string_                                                                                              |
+| `lastname`      | Admin user lastname                        | Doe                          | _string_                                                                                              |
+| `password`      | Admin user password                        | Correct Horse Battery Staple | _string_                                                                                              |
+| `email`         | Admin user email                           | pub@prestashop.com           | _string_                                                                                              |
+| `license`       | Show PrestaShop license after installation | 0                            | 0, 1                                                                                                  |
+| `theme`         | Theme name to install                      | "" (classic)                 | Theme name (located in `/themes`)                                                                     |
+| `ssl`           | Enable SSL (from PS 1.7.4)                 | 0                            | 0, 1                                                                                                  |
+| `rewrite`       | Enable rewrite engine                      | 1                            | 0, 1                                                                                                  |
+| `fixtures`      | Install fixtures                           | 1                            | 0, 1                                                                                                  |
+| `modules`       | Modules to install, separated by comma     | []                           | _array_ of module names (located in `/modules`)                                                       |
```

Comment:
What if we leave it blank? It will install all the modules, right?

Reply:
yes, i added a precision